usecases/ast_eval/evaluate/eval_string_starts_ends_with.go

```diff
@@ -0,0 +1,86 @@
+package evaluate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/cockroachdb/errors"
+
+	"github.com/checkmarble/marble-backend/models/ast"
+)
+
+type StringStartsEndsWith struct {
+	Function ast.Function
+}
+
+func NewStringStartsEndsWith(f ast.Function) StringStartsEndsWith {
+	return StringStartsEndsWith{
+		Function: f,
+	}
+}
+
+func (f StringStartsEndsWith) Evaluate(ctx context.Context, arguments ast.Arguments) (any, []error) {
+	leftAny, rightAny, err := leftAndRight(arguments.Args)
+	if err != nil {
+		return MakeEvaluateError(err)
+	}
+	if leftAny == nil || rightAny == nil {
+		return nil, nil
+	}
+
+	left, err := adaptArgumentToString(leftAny)
+	if err != nil {
+		return MakeEvaluateError(err)
+	}
+
+	rightString, errString := adaptArgumentToString(rightAny)
+	rightList, errList := adaptArgumentToListOfStrings(rightAny)
+
+	if errString == nil {
+		return startsEndsWithString(left, rightString, f.Function)
+	} else if errList == nil {
+		return startsEndsWithListOfStrings(left, rightList, f.Function)
+	} else {
+		return MakeEvaluateError(errors.Wrap(
```

Comment:
small syntax sugar: you can use "errors.Wrapf(..." and avoid the fmt.Sprintf inside